### DISCUSSION TITLE
Re-open a meeting or activity

### DISCUSSION THREAD

**Discussion by jordanh** [ParabolInc/parabol#6143] [New features]
A user writes:

>  ...we were running the retro you're referring to, and had just finished our voting round - I hit next, and it just... ended the meeting. Because I couldn't find a way to undo that, I was just kinda stuck finishing the retro without seeing what people upvoted and without the ability to create tasks and add notes to the reflections.
>
> So I mean, good news on your end, we didn't give up on Parabol or anything, we quite like it, in fact. Just, you know... it'd be nice to reopen a retro if it ends somehow? We've had probably about a dozen successful project retros on your platform so far, and everyone is really enjoying the format, the one you're referencing just kinda... I guess errored out.

Every once in a while we hear that folks have ended a meeting prematurely, either it was user error or clumsy UX on our part. Having a "no backsies" policy on an ended meeting can feel rather bad. Should we change the product to behave differently instead?

If we allow folks to re-open meetings, who can do it?
Ca you re-open all activities, or only those for which a meeting summary was not generated?
What corner cases would we need to consider?

**Comment by boylie3d** [comment]
I was the user that sent this ^ - if the issue is that I misclicked, that's totally on me, but I would say that either a prompt to warn users they're about to end the meeting that they need to confirm before it does it, or a way to undo the closing of the meeting within a short window. Or even move the "end meeting" button to be further away from the one you click most during the retro, the "Next" button. Any of those would be amazing, and would totally solve this problem, at least as far as I've experienced!

Thanks @jordanh for the prompt response by the way!

**Reply by tianrunhe** [reply]
Hi @boylie3d , thanks for raising this question to us! Today to end a meeting you have to click the button twice and there's indeed a prompt to warn user:

<img width="440" alt="Screen Shot 2022-03-01 at 10 42 22 AM" src="https://user-images.githubusercontent.com/1879975/156229471-be02e922-c83d-425a-b324-e26efd637119.png">

Therefore I was wondering if there's anything else weird happened to your meeting. Do you still have the meeting link you can share? Or the email of your account and when did you have the meeting? I'll definitely investigate and get to the bottom of the issue. Thanks again!

**Reply by boylie3d** [reply]
Hey @tianrunhe, thanks for the hand - I definitely thought it wanted you to click the button twice to end, that's why I was kinda convinced something had happened.

This was the meeting that got cut short - https://action.parabol.co/new-summary/bgqfGj1g0o, I ran the retro, and my email for Parabol is dave@globacore.com

Thanks for taking a look!

**Reply by tianrunhe** [reply]
Hey Dave, I've taken a look at all our logs and metrics, well, to computers it looks like you committed the double click on that end meeting button 😂, i.e., it's a valid meeting closure. The other way to end a meeting is through hot keys of cheat code, which, if it just happened to be the case you had, it would be the most amazing thing ever.

I'm sorry I couldn't root cause it but I do agree having a way to re-open a meeting will be helpful. Leaving this discussion open for our devs & designs.

Thanks again for using our product! Please keep these feedback coming!
